master/internal/routes.go

```diff
@@ -0,0 +1,32 @@
+package internal
+
+import (
+	"net/http"
+
+	"github.com/labstack/echo"
+)
+
+type redirect struct {
+	src    string
+	dest   string
+	method string
+	code   int
+}
+
+var redirects = [...]redirect{
+	{
+		src:    "/",
+		dest:   "/det",
+		method: http.MethodGet,
+		code:   http.StatusMovedPermanently,
+	},
+}
+
+func setupEchoRoutes(m *Master) {
+	for _, aRedirect := range redirects {
+		aRedirect := aRedirect
+		m.echo.Router().Add(aRedirect.method, aRedirect.src, func(c echo.Context) error {
+			return c.Redirect(aRedirect.code, aRedirect.dest)
+		})
+	}
+}
```

Comment:
```suggestion
func setupEchoRoutes(m *Master) {
	for idx := range redirects {
		func(r redirect) {
			m.echo.Router().Add(r.method, r.src, func(c echo.Context) error {
				return c.Redirect(r.code, r.dest)
			})
		}(redirects[idx])
	}
}
```

Reply:
hmm is the suggestion to create and execute a lambda function to avoid doing 
`aRedirect := aRedirect` ?

Reply:
there is a deterministic performance hit with creating an extra lambda each time right? but it would be negligible and if this pattern is more readable and common we should do this.

Reply:
it's negligible for sure, especially since this is run once on startup. it's fine either way, this isn't one i'll argue about.

Reply:
I updated this.

Reply:
https://github.com/determined-ai/determined/pull/1422/commits/3f396c9d4037359b18d7493f63483d7d9a8fa231